E/EMpht/Compat.toml

```diff
@@ -0,0 +1,2 @@
+[0]
+julia = "1.0.0-*"
```

Comment:
Can you use `julia = "1"` instead in your project file?

Reply:
Yes of course, I'll do that and re-register now. Thanks.